internal/services/kusto/kusto_cluster_resource.go

```diff
@@ -108,6 +128,24 @@ func resourceKustoCluster() *pluginsdk.Resource {
 				},
 			},
 
+			"allowed_fqdn_list": {
```

Comment:
i don't think we need list here?
```suggestion
			"allowed_fqdn_list": {
```
```suggestion
			"allowed_fqdns": {
```